test/components/NavBar/NavbarMobile.test.jsx

```diff
@@ -6,15 +6,15 @@ import NavbarMobile from '../../../src/components/NavBar/NavbarMobile';
 import createMatchMedia from '../../helpers/createMatchMedia';
 
 it('renders hamburger menu when on mobile', () => {
-  const { queryByLabelText, queryByRole } = render(
+  const { queryByRole } = render(
     <BrowserRouter>
       <NavbarMobile />
     </BrowserRouter>
   );
 
   const logo = queryByRole('img', { name: /logo$/ });
-  const navLinks = queryByLabelText('navigation tabs');
-  const hamburgerMenu = queryByLabelText('mobile menu');
+  const navLinks = queryByRole('tablist');
```

Comment:
Same question here... curious where this role is found - or if it's a default of MUI tabs?

Reply:
Yeah this is part of the [React tabs MUI component](https://mui.com/material-ui/react-tabs/). I was digging around looking to see if roles were applied and found it with the accessibility inspector in Firefox.

<img width="1392" alt="Screen Shot 2023-11-14 at 7 31 41 PM" src="https://github.com/codeforpdx/PASS/assets/65684361/5ce21ec8-b506-4b10-9d38-4a35fd1bbc74">